### DISCUSSION TITLE
JSON-LD Frame for Linked Places Format

### DISCUSSION THREAD

**Discussion by rybesh** [LinkedPasts/linked-places-format#31] [Ideas]
It would be very handy to have a [JSON-LD frame](https://www.w3.org/TR/json-ld11-framing/) that could be used to turn RDF into JSON-LD that adheres to the LPF. This is also something I’d be interested in working on. I’ve created some frames recently that produce valid GeoJSON. One of the questions this raises is “what is the best way to represent geometries in RDF?” to which I’ve concluded that the best answer is “as [JSON literals](https://www.w3.org/TR/json-ld11/#json-literals)” (which is now how GeoJSON-LD does it).

**Comment by kgeographer** [comment]
I will be working on this fall on creating one or more tools/scripts for converting between LPF and other formats, including RDF. This will be in collaboration with @rlzijdeman of IISH and KNAW in the Netherlands. We would be glad to have you join us!

When LP format was initially created, JSON-LD was at v1.0 and coordinates were not supported. There was little in the way of tools/libraries for processing JSON-LD, but we (@rsimon and I) plowed ahead assuming the situation would improve. It has, and so it is (past?) time to close this loop!
